test/AppNotificationTests/AppNotifications.Test.h

```diff
@@ -156,13 +159,13 @@ namespace AppNotificationHelpers {
                     getAllAsync.Cancel();
                 });
 
-            VERIFY_ARE_EQUAL(getAllAsync.wait_for(std::chrono::milliseconds(500)), winrt::Windows::Foundation::AsyncStatus::Completed);
+            VERIFY_ARE_EQUAL(getAllAsync.wait_for(c_timeout), winrt::Windows::Foundation::AsyncStatus::Completed);
             scopeExitGetAll.release();
 
             auto actualToastVector{ getAllAsync.GetResults() };
             if (actualToastVector.Size() == expectedToastVector.size())
             {
-                for (int actualToastIndex{ 0 }; (uint32_t)actualToastIndex < actualToastVector.Size(); actualToastIndex++)
+                for (int actualToastIndex{ 0 }; static_cast<uint32_t>(actualToastIndex) < actualToastVector.Size(); actualToastIndex++)
```

Comment:
Nit: I wonder if you could have initialized actualToastIndex as an uint32_t and avoid casting. No need to change that 😊.

But I see the `for` below doesn't have any cast, so why casting at all?

Reply:
Since vector::size() is type size_t, we have to use uint32_t. Fixed.